### PR TITLE
Add unit basic unit test for AppStateValue

### PR DESCRIPTION
**Pull request by MishaVeldhoen**
Hi @captKuk, here's a pretty basic unit test for the `AppStateValue`. Let me know if you have comments or want to discuss!

### CODE REVIEW

**Comment by captKuk** [comment]
Hi @MishaVeldhoen I just realised I didn't submit the review yet. It all looks fine except for the error message in [this commit](https://github.com/materialscloud/mz-bokeh-package/commit/eadd2aaf464c9fe7d8aa97e461ffa90db65c421f) and maybe consider using the method inspect.ismethod for determining the number of args of the callback function.

**Review by captKuk — CHANGES_REQUESTED** [review]
Please see my comments...

File: tests/unit/test_app_state_value.py [hunk 1]
```diff
@@ -27,38 +30,69 @@ def dummy_callback_one_arg(new_value: Any) -> None:
...
-    def callback_no_args(self) -> None:
+    def method_no_args(self) -> None:
         pass
 
-    def callback_one_arg(self, new_value) -> None:
+    def method_one_arg(self, new_value: Any) -> None:
         self.values.append(new_value)
 
+    def method_two_args(self, new_value: Any, other_value: Any) -> None:
+        self.values.append(new_value)
```
Comment (captKuk) [inline tests/unit/test_app_state_value.py:40]: Will the test ever use the appended _new\_value_ in the case of `method_two_args`? Leaving the line `self.values.append(new_value)` here, might evoke the impression there is a valid use-case for that, right?

**Review by captKuk — COMMENTED** [review]
Looks fine to me now. There is only one nit-picky question about the `method_two_args`. Apart from that it's ok and ready to be merged.

File: tests/unit/test_app_state_value.py (hunk 1 above)
Comment (MishaVeldhoen) [reply tests/unit/test_app_state_value.py]: You're right! I'll fix it.

**Pull request merged**